pkg/oc/admin/diagnostics/diagnostics/cluster/aggregated_logging/diagnostic_test.go

```diff
@@ -1,8 +1,9 @@
 package aggregated_logging
 
 import (
-	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/log"
 	"testing"
+
+	"github.com/openshift/origin/pkg/oc/admin/diagnostics/diagnostics/log"
```

Comment:
how do you do these re-sortings automatically?

Reply:
> how do you do these re-sortings automatically?

probably goimports.  the editor does something.